plugins/inventory/nb_inventory.py

```diff
@@ -365,6 +365,7 @@ def group_extractors(self):
             "disk": self.extract_disk,
```

Comment:
Do you mind updating the inventory related tests to add the status for each device after you fix the one question below?

The inventory tests are shown in the failed tests. I should be done with fixing the tests tonight as well if you want to rebase off devel once I let you know and then just fix the failing inventory tests from there.

Reply:
Sorry, I'm a newbie. I don't know, how to work with this test framework. :-(

Reply:
You'll need to look at what failed in the Travis CI run and then correct the corresponding files for the inventory tests. The tests will tell you what each file is expecting.

Those are all stored in `tests/integration/targets/inventory/files`